tests/Common/TokensTest.php

```diff
@@ -98,6 +98,65 @@ public function testTokenProperties()
     {
         $currentToken = $this->_client->verifyToken();
 
+        // Master token should not return unencrypted token
+        $this->assertArrayNotHasKey('token', $currentToken);
```

Comment:
Asi bych tady doplnil assert že ten projekt má přiřazenou featuru na hide tokenu?

Reply:
Akorát to teda bude assert nad mockem :) Ale doplnit to můžu.

Reply:
Jo, moment, to je jiný test. To bych měl...

Reply:
Udělal jsem tam podmínku, aby to vždycky kontrolovalo podle toho, jak jsou nastavené ty featury.